### PR TITLE
M707 Vulture AMR: Now actually available edition 

### PR DESCRIPTION
**Pull request by Rem1-code**
# About the pull request
Req now starts with an M707 Vulture case, that I've hopefully popscalded correctly so it only spawns in very low numbers on highpop

# Explain why it's good for the game
The M707 is a unique and complex weapon that is in many ways a tool for what CM aims to achieve. It requires intense teamwork, lots of strategy, but when used correctly, it's very rewarding. It adds a new dynamic to the battlefield, and still has the mechanical complexity and drawbacks to allow it to fit perfectly without disrupting balance. Such a weapon should not go to waste. 

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

image

</details>

# Changelog
:cl: Rem1
add: the M707 Vulture AMR to requisitions. 
/:cl:

### CODE REVIEW

**Comment by efzapa** [comment]
Detroit: Become Friendly Fire

**Comment by HeresKozmos** [comment]
To my knowledge this is event-only, why should it be made available every round?

**Comment by Rem1-code** [comment]
> To my knowledge this is event-only, why should it be made available every round?

As far as I know there was never anything specific intending it to be event only, and it only ended up being so for ease of implementation. I could be wrong.

It's a fully fleshed out gun that was originally intended to be available, that uses some very complex and interesting mechanics. I don't think it needs to be event exclusive.

**Comment by Tired-Medic** [comment]
More additions mean less stale of a game. If it works badly or too many people cry about it we can always remove it. Would give it a try for a week so everyone can test it out.

**Review by Zonespace27 — COMMENTED** [review]
No comment on if it should be added to the gameplay loop at all, but if it is, it should definitely be locked to 1 instead of scaling

**Comment by efzapa** [comment]
I should point out that this should not be what seems to be a free spec gun that can just be vended for *anyone*

**Comment by ihatethisengine** [comment]
Please learn that getting obliterated from the outside of your screen with no actual way to anticipate and thus counter is not fun. Sniper AMR highlighted this perfectly, so please don't add the gun that literally one shots everything. And yeah, we don't need to find out how bad for the game it is, we already know this from the many occurrences when it was spawned by admins. It was very bad.

**Comment by Rem1-code** [comment]
> Please learn that getting obliterated from the outside of your screen with no actual way to anticipate and thus counter is not fun. Sniper AMR highlighted this perfectly, so please don't add the gun that literally one shots everything. And yeah, we don't need to find out how bad for the game it is, we already know this from the many occurrences when it was spawned by admins. It was very bad.

image



More seriously, CM has many far more lethal weapons that are wayyyy easier to use. I think that in wasn't that bad at all in the TMs I saw, but I acknowledge that I could be wrong as always.

**Comment by efzapa** [comment]
> > Please learn that getting obliterated from the outside of your screen with no actual way to anticipate and thus counter is not fun. Sniper AMR highlighted this perfectly, so please don't add the gun that literally one shots everything. And yeah, we don't need to find out how bad for the game it is, we already know this from the many occurrences when it was spawned by admins. It was very bad.
> 
> image
> 
> More seriously, CM has many far more lethal weapons that are wayyyy easier to use. I think that in wasn't that bad at all in the TMs I saw, but I acknowledge that I could be wrong as always.

i think this is what the kids call "powercreep"

**Comment by VileBeggar** [comment]
there's a holotargetting version of the vulture that is a lot less lethal and requires friendly marines to take advantage of it. you might have better luck adding that in than an offscreen instakill weapon (this is only half a shameless plug)

**Comment by Rem1-code** [comment]
> > > Please learn that getting obliterated from the outside of your screen with no actual way to anticipate and thus counter is not fun. Sniper AMR highlighted this perfectly, so please don't add the gun that literally one shots everything. And yeah, we don't need to find out how bad for the game it is, we already know this from the many occurrences when it was spawned by admins. It was very bad.
> > 
> > 
> > More seriously, CM has many far more lethal weapons that are wayyyy easier to use. I think that in wasn't that bad at all in the TMs I saw, but I acknowledge that I could be wrong as always.
> 
> i think this is what the kids call "powercreep"

I don't know what you want to me say about that. I don't think new content should be considered powercreep, but to each their boxstation I guess.

**Comment by TheManWithNoHands** [comment]
> Please learn that getting obliterated from the outside of your screen with no actual way to anticipate and thus counter is not fun. Sniper AMR highlighted this perfectly, so please don't add the gun that literally one shots everything. And yeah, we don't need to find out how bad for the game it is, we already know this from the many occurrences when it was spawned by admins. It was very bad.

Yes, it really isnt fun if your suddently oneshot by one of your teammates. It has a higfher marine killcount then xenos Iirc. Being that it has no IFF. It has also features that basicly taddle on the sniper team (you need an actual competent spotter for this to work) and roughly where they are Iirc.
The vulture really isnt OP, its just that its used so little that xenos didnt have the time to learn how to figth it. Thats all. I mean, how do you expect xenos to figure out how to go against a tool that is ingame once every blue moon? Most dont read PRs or forums or wiki.

**Comment by TheManWithNoHands** [comment]
> How do you fight a gun you cannot even see? You cannot be serious.

The gun has multiple warnings/tell tells for xenos when its fired, basicly announcing its position every time you take a shot.
One it has a uniqe loud firing sound, and it also gives xenos a large red text message about it.
You cannot use the Vulture from behind marine lines, you need to go to the side line where your unprotected.

You can know where the Vulture is, it doesnt one shot, and it only works if the users expose themself and are out of cover. 
If you cant defeat a tool with so many weaknesses, its a skill issue.

>When there is a hedge ravager with that big blue shield indicating that you shouldn't shoot it, marine mains be like: EWWW SO UNFAIR
When there is an one-shot sniper rifle you cannot see as a xeno, marine mains be like: erm just learn to counter it somehow

The Rav shield is a powerfull tool for xenos that posses an extremly ovious tell tail, allowing marines to easily adapt to it or get punished.
The Vulture is a powerfull tool for marines that posses an extremly obvious tell tail, allowing xenos to easily adapt to it or get punished.

Kettle calling the pot black.

**Comment by Butlerblock** [comment]
please stop countering criticism by justifying it based on a different mechanic that is related in similarities alone. nobody cares about boiler, we're talking about vulture here, if you have a problem with boiler, go complain somewhere else.

no you can't justify offscreen insane burst damage by relating it to a xeno you can just walk away from. hedge has design flaws, and should not be replicated in even more unhealthy designs

**Comment by ihatethisengine** [comment]
> You cannot use the Vulture from behind marine lines, you need to go to the side line where your unprotected.

You clearly haven't played a single round with vulture. Hello, admins were spawning it even on hijack and people were one shoting xenos in 3 tiles wide corridors. You can easily use it from behind marines lines, from cades, so you just made up this weakness to justify this gun that wasn't added to the game for a clear reason.

> The Vulture is a powerfull tool for marines that posses an extremly obvious tell tail

How are you so dense? It's **off-screen**, there is no warning before you are already doomed.

**Comment by TheManWithNoHands** [comment]
> You clearly haven't played a single round with vulture. Hello, admins were spawning it even on hijack and people were one shoting xenos in 3 tiles wide corridors. You can easily use it from behind marines lines, from cades, so you just made up this weakness to justify this gun that wasn't added to the game for a clear reason.

"A gun with no IFF that one tabs marines can be easily used behind marine lines" -ihatethisengine 2024

**Comment by deathrobotpunch** [comment]
got given this as a mini event as a CL for testing. the poor corpsman we tested on broke every single bone in their body and had the maximum amount of brute possible.

you might wanna change it an itty bit

**Comment by TheManWithNoHands** [comment]
> got given this as a mini event as a CL for testing. the poor corpsman we tested on broke every single bone in their body and had the maximum amount of brute possible.

The ammount of FF was very much intentional by Zone & Thwomper Iirc

File: code/game/machinery/vending/vendor_types/requisitions.dm [hunk 1]
```diff
@@ -40,6 +40,7 @@
 		list("M56D Heavy Machine Gun", floor(scale * 2), /obj/item/storage/box/guncase/m56d, VENDOR_ITEM_REGULAR),
 		list("M2C Heavy Machine Gun", floor(scale * 2), /obj/item/storage/box/guncase/m2c, VENDOR_ITEM_REGULAR),
 		list("M240 Incinerator Unit", floor(scale * 2), /obj/item/storage/box/guncase/flamer, VENDOR_ITEM_REGULAR),
+		list("M707 Anti Material Rifle", floor(1), /obj/item/storage/box/guncase/vulture, VENDOR_ITEM_REGULAR),
```
Comment (Zonespace27) [inline code/game/machinery/vending/vendor_types/requisitions.dm:43]: Why are you flooring 1

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/game/machinery/vending/vendor_types/requisitions.dm (hunk 1 above)
Comment (Rem1-code) [reply code/game/machinery/vending/vendor_types/requisitions.dm:43]: because I don't know how to code and just removed the scaling :(

will attempt to fix

Comment (Rem1-code) [reply code/game/machinery/vending/vendor_types/requisitions.dm]: fixed

**Comment by realforest2001** [comment]
Needs balance review, Vulture wasn't balance checked as it wasn't in the game loop.

**Comment by TheManWithNoHands** [comment]
Oh yeah, Iirc there is a less hardcore version of the Vulture that has acces to Holo-ammo. That migth be a better for regular gameplay. dont remember who made it, Vile maybe.

**Comment by morrowwolf** [comment]
> > To my knowledge this is event-only, why should it be made available every round?
> 
> 
> 
> As far as I know there was never anything specific intending it to be event only, and it only ended up being so for ease of implementation. I could be wrong.
> 
> 
> 
> It's a fully fleshed out gun that was originally intended to be available, that uses some very complex and interesting mechanics. I don't think it needs to be event exclusive.

It was very specifically added with the caveat it would remain event only.

The vulture is really cool.

The vulture is horrendously unfun for anyone that has to exist around it.

Balance is not really relevant. The design of the gun in the CM standard round is the problem.

**Comment by Rem1-code** [comment]
> > > To my knowledge this is event-only, why should it be made available every round?
> > 
> > 
> > As far as I know there was never anything specific intending it to be event only, and it only ended up being so for ease of implementation. I could be wrong.
> > It's a fully fleshed out gun that was originally intended to be available, that uses some very complex and interesting mechanics. I don't think it needs to be event exclusive.
> 
> It was very specifically added with the caveat it would remain event only.
> 
> The vulture is really cool.
> 
> The vulture is horrendously unfun for anyone that has to exist around it.
> 
> Balance is not really relevant. The design of the gun in the CM standard round is the problem.

Hi,

I find in my experience CM is mainly balanced on giving both sides a lot of things that are fun to *play*, at the cost of them possibly being unfun to play *against*.

This can be seen by the existence of runner, lurker, warrior, hedge ravager, boiler, queen, OT, stims, greenos, shotgun buck PBs, lesser drones, playable facehuggers, OBs, CAS, Mortar, SADAR, GL, MOU, nuke, xeno king, and turrets.

Thus, I don't think this is a relevant issue unless a major rework of CM is done to make things both fun to play and fun to play against. Right now, I think CM is mainly focused on indirect counters that aren't equivalent. The xenomorph counterpoint to the nuke, the king, is arguably incomparable. Both aren't fun to play against. Both are fun to play as/with. 

regards

**Comment by morrowwolf** [comment]
You're wrong.

Regards

**Comment by Rem1-code** [comment]
i am once again persecuted by the neverplayer mafia

**Comment by TheManWithNoHands** [comment]
> Hi,
> 
> I find in my experience CM is mainly balanced on giving both sides a lot of things that are fun to _play_, at the cost of them possibly being unfun to play _against_.

Im sad to disapoint you, but nah. CM isnt balanced by some genuises that know what they are doing. Everyone here is some uniqe flavor of dumb. Most of the stuff thats unfun to play against is either blown out of proportion because of salt, or just a fuck up.
In general the goal in pvp is always to make tools that are fun to figth with, and fun to figth against. Just thats hard af to do that.

On that note, dont forget the brainstorming we had on discord Rem1. Current Vulture sucks to figth against, but dont forget we came up with a rough idea how to make it at least ok. Its more worth it to put your effort into that, then arguments on git.

**Comment by Rem1-code** [comment]
> > Hi,
> > I find in my experience CM is mainly balanced on giving both sides a lot of things that are fun to _play_, at the cost of them possibly being unfun to play _against_.
> 
> Im sad to disapoint you, but nah. CM isnt balanced by some genuises that know what they are doing. Everyone here is some uniqe flavor of dumb. Most of the stuff thats unfun to play against is either blown out of proportion because of salt, or just a fuck up. In general the goal in pvp is always to make tools that are fun to figth with, and fun to figth against. Just thats hard af to do that.
> 
> On that note, dont forget the brainstorming we had on discord Rem1. Current Vulture sucks to figth against, but dont forget we came up with a rough idea how to make it at least ok. Its more worth it to put your effort into that, then arguments on git.

Er. If it's blown out of proportion because of salt or a fuck up, then it's still unfun to play against. CM has never adopted a 2 way strategy and we're far off from that now. This can easily be seen by all the previous example. Nobody enjoys playing against boiler, or playing against SADAR rockets. Instead people enjoy both playing boiler and playing SADAR with OT rockets. This has always been the case and unless we take a major developmental shift (which I think would be good but that's a conversation for another time) CM will remain as such.

**Comment by TheManWithNoHands** [comment]
> unless we take a major developmental shift (which I think would be good but that's a conversation for another time) CM will remain as such.

Then help start with that. Help make stuff more fun to figth against. Just saying "i dont need to make it fun to figth against, because other stuff isnt" aint a solution. Its just an excuse. If we want the game to be more fun for everyone we have to change it to that. Piece by piece.

So if you want CM to be more fun for both sides, the attacker and the attacked, help do that.
Im myself working on tank balance stuff for that, myself. Thats not something everyone decides together at the same moment.
Its something done bit by bit.

If you dont want to, thats fine. But dont say stuff like you dont do it because xyz isnt like that. Especially if you can activly help make the game more fun instead.

**Comment by Rem1-code** [comment]
> > unless we take a major developmental shift (which I think would be good but that's a conversation for another time) CM will remain as such.
> 
> Then help start with that. Help make stuff more fun to figth against. Just saying "i dont need to make it fun to figth against, because other stuff isnt" aint a solution. Its just an excuse. If we want the game to be more fun for everyone we have to change it to that. Piece by piece.
> 
> So if you want CM to be more fun for both sides, the attacker and the attacked, help do that. Im myself working on tank balance stuff for that, myself. Thats not something everyone decides together at the same moment. Its something done bit by bit.
> 
> If you dont want to, thats fine. But dont say stuff like you dont do it because xyz isnt like that. Especially if you can activly help make the game more fun instead.

Ares when you convince the entire maintainer team and the host to drastically change the way the game currently operates and code a full rework of runner, lurker, warrior, hedge ravager, boiler, queen, OT, stims, greenos, shotgun buck PBs, lesser drones, playable facehuggers, OBs, CAS, Mortar, SADAR, GL, MOU, nuke, xeno king, and turrets, among others

I think making everything good and fun to play is a far better solution then the balance hell

**Comment by ihatethisengine** [comment]
> I think making everything good and fun to play is a far better solution then the balance hell

This is a competitive game, if you just gonna add some random stuff with little to no thought of how it will affect the game, you will only make it more frustrating for everyone. Vulture should stay buried. I don't wanna get DELETED off screen by Vulture with no warning, just like I don't wanna get 6 fracs from FF because my teammate got his hands on the gun.

CM balance is far from perfect, but that's no reason to just throw everything in because of your subjective "fun". Everything you mentioned has its problems, but it doesn't compare to Vulture. King is ok to play against. The only bs ability he has is his ability to spam shove marines by walking over them. It's pretty easy to counter though by keeping your distance. And remember, King is an end game mechanic. Whereas with Vulture you just drop it in req for some reason. You don't even add another stupid sniper spec option, you just leave it in req. Not like we need more stupid sniper specs options.

Honestly I am baffled this is not closed yet. LTB was told a hard "no" and for good reason. But is anyone among devs really thinking about adding Vulture???

**Comment by Rem1-code** [comment]
> > I think making everything good and fun to play is a far better solution then the balance hell
> 
> This is a competitive game, if you just gonna add some random stuff with little to no thought of how it will affect the game, you will only make it more frustrating for everyone. Vulture should stay buried. I don't wanna get DELETED off screen by Vulture with no warning, just like I don't wanna get 6 fracs from FF because my teammate got his hands on the gun.
> 
> CM balance is far from perfect, but that's no reason to just throw everything in because of your subjective "fun". Everything you mentioned has its problems, but it doesn't compare to Vulture. King is ok to play against. The only bs ability he has is his ability to spam shove marines by walking over them. It's pretty easy to counter though by keeping your distance. And remember, King is an end game mechanic. Whereas with Vulture you just drop it in req for some reason. You don't even add another stupid sniper spec option, you just leave it in req. Not like we need more stupid sniper specs options.
> 
> Honestly I am baffled this is not closed yet. LTB was told a hard "no" and for good reason. But is anyone among devs really thinking about adding Vulture???

CM is not a TDM. This has been made very clear. All of these things apply to plenty of other features in the game. Nobody wants to die to a an AP SADAR rocket, yet we don't remove them because they aren't fun to play against. 

King is one of many, many examples. the vulture is a net positive as it's fun for 2 players, as opposed to usually where CM is focused on stuff exclusive to a single player. I view it in the the same way one would view the Mortar. It's exceptionally hard to miss someone using it given it gives you their exact position when fired, very hard to use, and even easier to counter then the mortar. idk what you want me to tell you, sniper spec already has 2 guns 

This isn't the tank

**Comment by ihatethisengine** [comment]
> CM is not a TDM

Sure.



> Nobody wants to die to a an AP SADAR rocket, yet we don't remove them because they aren't fun to play against.

At least SADAR is on your screen and you can react to it before you get shot. 


> the vulture is a net positive as it's fun for 2 players

It's not net positive if it's frustrating for literally everyone else. King is not as frustrating to fight against as a marine and doesn't one shot fellow xenos.

**Comment by Rem1-code** [comment]
> > CM is not a TDM
> 
> Sure. 
> 
> > Nobody wants to die to a an AP SADAR rocket, yet we don't remove them because they aren't fun to play against.
> 
> At least SADAR is on your screen and you can react to it before you get shot. It's balanced by many other things such as additional wield delay. And guess what? It had a scope. And it was removed because of balance.
> 
> > the vulture is a net positive as it's fun for 2 players
> 
> It's not net positive if it's frustrating for literally everyone else. King is not as frustrating to fight against as a marine and doesn't one shot fellow xenos.

SRPBTDM vs TDM

Can you react to it? Not always. And that wasn't the question, the question was, is it fun.

Almost everything in CM is fun for one person, the user. Vulture, by virtue of having 2 users is one of the rare exceptions, along with perhaps fire support. (GP/SO/Mortar user and spotter)

**Comment by ihatethisengine** [comment]
> SRPBTDM vs TDM

Still TDM, notice how there is no "erm some subjective fun that I personally define" stuff in SRPBTDM? But there is TDM, which makes the game competitive, just like I said. So balance actually matters.

> Can you react to it? Not always. And that wasn't the question, the question was, is it fun.

More fun than Vulture for sure. You can play around it. It can be fun to be engaged in mind games with SADAR as a T3, baiting his shot to beat him up when he miss.

> Almost everything in CM is fun for one person, the user. Vulture, by virtue of having 2 users is one of the rare exceptions, along with perhaps fire support. (GP/SO/Mortar user and spotter)

Braindead take. Just like with SADAR, you can have fun playing against most things (not all of them, especially not off-screen damage), there are also people who work with you, who follow up on your shots, who cover you etc. Is screech (another pretty frustrating thing) only fun for the Queen? Not really, it mostly gives opportunity to deal damage for other xenos. And it can be engaging to bait a screech as a marine and then punish the Queen by rushing her. Also notice how it is also not off-screen and even has a warning sound (steps). I wonder why, maybe there is a reason?

Is this really your only justification? Vulture is good because two people use it? Should bring back old APC then.

**Comment by TheManWithNoHands** [comment]
> Almost everything in CM is fun for one person, the user.

I think your confusing the need for counter play with unfun to play against.
A LOT of competetive games derive their fun out of counterplay, because thats the crux of the gameplay. 
You have a tool, your enemys knows you have said tool and what it does. So they need to adjust their playstile to counter your tool. Wich requires you change your playstyle with said tool so their counter doesnt work properly anymore. Rainbow 6 is fully build around that idea. That and corner prefire.
Yes, we dont have a competetive mode or something. But as long as its PvP, its always gonna be competetive. Some rules in gaming cannot be broken. Like TTP.

We have a LOT of things as well that require you to change your playstyle to counter. Warrior has the ability to jump at you to stun you, and fling you. Problem is WAR cant tank very well. So they cant do it out in the open and have to do it from behind cover/near cover. That makes the situations they can use it limited. Furthermore, their jump hass a set range. If you keep your range, wich is easy because you have a gun, around corners, doors and in general blind spots you counter WAR entire kit.

You need to know the tools of the enemy, to counter it. A lot of stuff in CM works like that. A LOT.
Best example Hedghog shield. Shoot it, get punished. Dont shoot it, rav basicly nothing.
Yes, they could work better, but a lot of people are activly trying to make it better.
Saying you will walk two steps back in terms of balance because other stuff needs fixing, doesnt help anybody.
And it sure as feck wont help putting the gun in the game.

We talked on discord arleady how to make the gun more fun to figth against. Just do that and stop with this pointless shit.

**Comment by Rem1-code** [comment]
> > SRPBTDM vs TDM
> 
> Still TDM, notice how there is no "erm some subjective fun that I personally define" part in SRPBTDM? But there is TDM, which makes the game competitive, just like I said. So balance actually matters.
> 
> > Can you react to it? Not always. And that wasn't the question, the question was, is it fun.
> 
> More fun than Vulture for sure. You can play around it. It can be fun to be engaged in mind games with SADAR as a T3, baiting his shot to beat him up when he misses.
> 
> > Almost everything in CM is fun for one person, the user. Vulture, by virtue of having 2 users is one of the rare exceptions, along with perhaps fire support. (GP/SO/Mortar user and spotter)
> 
> Braindead take. Just like with SADAR, you can have fun playing against most things (not all of them, especially not off-screen damage, wonder why), there are also people who work with you, who follow up on your shots, who cover you etc. Is screech (another pretty frustrating thing) only fun for the Queen? Not really, it mostly gives opportunity to deal damage for other xenos. And it can be engaging to bait a screech as a marine and then punish the Queen by rushing her. Also notice how it is also not off-screen and even has a warning sound (steps). I wonder why, maybe there is a reason?
> 
> Is this really your only justification? Vulture is good because two people use it? Should bring back old APC then. It even shares some of the same problems with Vulture, by ruining the game for both marines and xenos.

What? You can perfectly choose to want CM to be a pure TDM, but this isn't TGMC and I think that's been made very clear. I had multiple conversations to make sure I understood CMs design philosophy and was confirmed in my beliefs. 

"It can be fun to engage in mind games with SADAR as a T3." Replace SADAR with Vulture then, you've found your way to enjoy it. I find this even funnier given you seem to hate OT rockets. I thought it was fun? 

The Vulture warns you when a shot is fired as well. You've admitted to never having used it or played while it was being TMed so this take feels naive. This APC comment also fails to understand part of the reason APC was removed was because it was unfun to play.

Given you haven't played against or with Vulture at all, and seem to insist on a vision for CM that's not being pursued, I think I will quote our esteemed ex head maint:

You're wrong.

Regards

**Comment by Rem1-code** [comment]
> > Almost everything in CM is fun for one person, the user.
> 
> I think your confusing the need for counter play with unfun to play against. A LOT of competetive games derive their fun out of counterplay, because thats the crux of the gameplay. You have a tool, your enemys knows you have said tool and what it does. So they need to adjust their playstile to counter your tool. Wich requires you change your playstyle with said tool so their counter doesnt work properly anymore. Rainbow 6 is fully build around that idea. That and corner prefire. Yes, we dont have a competetive mode or something. But as long as its PvP, its always gonna be competetive. Some rules in gaming cannot be broken. Like TTP.
> 
> We have a LOT of things as well that require you to change your playstyle to counter. Warrior has the ability to jump at you to stun you, and fling you. Problem is WAR cant tank very well. So they cant do it out in the open and have to do it from behind cover/near cover. That makes the situations they can use it limited. Furthermore, their jump hass a set range. If you keep your range, wich is easy because you have a gun, around corners, doors and in general blind spots you counter WAR entire kit.
> 
> You need to know the tools of the enemy, to counter it. A lot of stuff in CM works like that. A LOT. Best example Hedghog shield. Shoot it, get punished. Dont shoot it, rav basicly nothing. Yes, they could work better, but a lot of people are activly trying to make it better. Saying you will walk two steps back in terms of balance because other stuff needs fixing, doesnt help anybody. And it sure as feck wont help putting the gun in the game.
> 
> We talked on discord arleady how to make the gun more fun to figth against. Just do that and stop with this pointless shit.

CM doesn't derive it's fun out of counterplay. Waiting 5s to shoot the hedgehog ravager is not fun. If you wish to fully rework the game to do so, let me know.

**Comment by TheManWithNoHands** [comment]
> CM doesn't derive it's fun out of counterplay. Waiting 5s to shoot the hedgehog ravager is not fun. If you wish to fully rework the game to do so, let me know.

It is fun. Because you can use sait 5 seconds to do something. Like, trap the rav thats kinda just standing there. Shoot the high priority target that the rav is trying to take the attention off from.

There are a lot of things you can do in those 5 seconds. All involve, ignoring the RAV. If it where in a vacum in a 1v1, then yes, it would be a bad desing. But its not. Its a mechanic meant to be utilized via teamwork. If the rav just blindly uses their shield and goes in, wont do much. But if they use it to direct attention away from their team while they do something, then it works very well.

Counterplay and teamwork. The cornerstones of team based competetive pvp. Yes, we are more silly goobers. But there is a reason to balance stuff around the high end comp scene in games. But that would take too long.

**Comment by Rem1-code** [comment]
> > CM doesn't derive it's fun out of counterplay. Waiting 5s to shoot the hedgehog ravager is not fun. If you wish to fully rework the game to do so, let me know.
> 
> It is fun. Because you can use sait 5 seconds to do something. Like, trap the rav thats kinda just standing there. Shoot the high priority target that the rav is trying to take the attention off from. Your not forced to do nothing in those 5 seconds. You only cant shoot a single target. Thats it. You can still shoot anything else, build, move, whatever that doesnt involve shooting one single target. Its bait after all. You can already do that without a mechanic in ANY game. Just, ignore it on focus on more important stuff that "shooty mc shootyface" over there. Not like they can do anything.
> 
> There are a lot of things you can do in those 5 seconds. All involve, ignoring the RAV. If it where in a vacum in a 1v1, then yes, it would be a bad desing. But its not. Its a mechanic meant to be utilized via teamwork. If the rav just blindly uses their shield and goes in, wont do much. (unless marines are dumb as fuck, wich isnt a thing you should concider in the design) But if they use it to direct attention away from their team while they do something, then it works very well.
> 
> Counterplay and teamwork. The cornerstones of team based competetive pvp. Yes, we are more silly goobers. But there is a reason to balance stuff around the high end comp scene in games. But that would take too long too talk about.

So the fun is...running away for 5 seconds from the shielded rav
Yeah I think im going to requote our ex maint head on this one

**Comment by TheManWithNoHands** [comment]
> > It is fun. Because you can use sait 5 seconds to do something. Like, trap the rav thats kinda just standing there. Shoot the high priority target that the rav is trying to take the attention off from. 

> So the fun is...running away for 5 seconds from the shielded rav Yeah I think im going to requote our ex maint head on this one

How can you missquote something that badly? I didnt even say run away. Just shoot the other xenos. The rav aint stopping you from doing that.

**Comment by Nanu308** [comment]
Needs to be tweaked extensively before it's ready for normal gameplay.

**Comment by Nanu308** [comment]
Since stalebot is deactivated gonne close this manually

**Pull request closed without merging**